examples/tests/invalid/immutable_container_types.test.w

```diff
@@ -1,33 +1,33 @@
 let m1 = Map<str>{"a" => "hi"};
 
 m1.set("a", "bye");
-// ^^^ Unknown symbol "set" (TODO: better error message https://github.com/winglang/wing/issues/1660)
+// ^^^ Property "set" doesn't exist in "Map" (TODO: better error message https://github.com/winglang/wing/issues/1660)
```

Comment:
Since `set` refers to a method (a function value) and not a property (a non-function value) the message feels off to me here. Perhaps we can change the error based on whether it's a method or property?

Or we can fall back to using the general term "member"

Reply:
I think the general "member" is good enough here. I'll make the fix.